### ISSUE TITLE
API request failed 

### ISSUE THREAD

**Issue by SHLOK333** [themoritz/txgraph#20]
While running the model got this error API request failed!
Missing signature headers.
can you please help me out

![Screenshot 2024-09-15 233216](https://github.com/user-attachments/assets/81963225-0f14-4e6f-82c8-732c8ae39a5b)

**Comment by themoritz** [comment]
This is probably happening because you are accessing the API without the web client at https://txgraph.info (maybe developing locally?). I have restricted the API to only allow requests from the hosted web client, but if you are interested I can provide you with a token to access the API for your own development.

In this case, please send an email to hello@txgraph.info with your name, what you intend to use the API for, as well as rough usage patterns (like number of requests per day or week).

Cheers,
Moritz

**Comment by kakulukia** [comment]
Just found this issue because https://txgraph.info/ has the same problem. @themoritz are you aware of this and could you have a look?

**Comment by themoritz** [comment]
Yes, I'm aware of this, but currently don't have time to fix it. The server has run out of disk space because the blockchain has grown so much and I need to migrate the app to another server.

**Comment by themoritz** [comment]
@kakulukia this is now fixed, sorry it took so long for me to find time to do it.
